### DISCUSSION TITLE
Is there a way to reduce the amount of logging done?

### DISCUSSION THREAD

**Discussion by 0xabrar** [vercel/community#493] [Help] [answered]
Looking through my logs, it seems that every request made to a Vercel endpoint is logged. So every GET request is logged and then stored into my log drain. Is there a way for me to selectively choose what gets sent to my log drain (I'm using logtail)? I see that there's options for me to modify or enrich my data before it's sent off, but couldn't find docs for dropping items.

A large portion of these requests aren't that useful to see, and really just add noise.

**Comment by samcx** [comment, marked as answer]
Unfortunately, this is not possible. I think this should be the job of the Log Drain to properly filter all the requests that come back from Vercel. Are there no options to filter logs in Logtail?
